### ISSUE TITLE
Move to next flagged misspelling feature

### ISSUE THREAD

**Issue by Gene703122** [nvaccess/nvda#13323]
### Is your feature request related to a problem? Please describe.
At present, NVDA announces flagged misspelled words in documents.  However, the only programs that allow movement to the next flagged misspelled word use Browse Mode, as in Word.  There is no efficient way to move to the next misspelling in other programs such as in the Thunderbird message composition edit field or in any word processor that doesn't support browse mode.  

When I read a message before sending it, I read line by line.  If there is a misspelled word on a line, I have to move word by word to get to it, a very cumbersome procedure if the word is somewhat far into the line, as is often the case.  
### Describe the solution you'd like
There should be a command to move to the next flagged misspelling that works anywhere where NVDA recognizes flagged words.  
### Describe alternatives you've considered
I can use f7 but that means that if I want to avoid the annoyance of moving inefficiently to flagged misspelled words, that I have to perform two procedures, spell check the document and review it separately.  A move to next flagged word command would allow both review and efficient spell checking in one logical procedure.  Also, and more important, using f7 doesn't present an efficient way to see the word in context.  Reviewing a document and spell checking as you go does.  

Perhaps there should be a move to previous misspelled word command as well.  I saw that suggested on a list but I'm not sure how useful it would be.  It would be Goode if it were discussed.

**Comment by marrie** [comment]
a list might be feasible, or invoking the contextual menu on said word after curser is placed on said word might be more practical.

**Comment by lukaszgo1** [comment]
This is a duplicate of #13293

**Comment by josephsl** [comment]
Closing as flagged as a duplicate.

A sincere plea: please search for an issue before posting a new one. Thanks.
